### ISSUE TITLE
PDF Bookmarks

### ISSUE THREAD

**Issue by drsbaitso** [owncloud/ios#454]
Not sure if this is a feature request per say, but it would be nice to have bookmarks accessible in the PDF viewer.  That, or have an option in the settings to be able to set which app opens PDF or other document formats.  For instance, I want to open all PDFs in Adobe Reader so I get the bookmark functionality without having to hit three button presses to do the open in...

**Comment by ggdiez** [comment]
Hello @drsbaitso 

The library that we uses as pdf reader allow us to update the pdf reader to read bookmarks, but not for create new ones. We could be develop it.

I don't know if that could be enough for you?

The other solution that you say, have an option in settings...., is not possible, Apple does not allow know what others apps are installed in a device.

**Comment by drsbaitso** [comment]
@ggdiez Yeah, reading the bookmarks would be wonderful.  I have a couple users that would like to be able to jump between sections in their bigger documents and as of now I have him opening it with the open in command which works ok but if it were integrated into the library in the ownCloud client I think that would be great.
